pro/src/pages/AdageIframe/app/components/OffersInstantSearch/OffersSearch/Offers/Offers.tsx

```diff
@@ -166,7 +175,7 @@ export const OffersComponent = ({
               userEmail={userEmail}
               userRole={userRole}
             />
-            {index === 1 && isSatisfactionSurveyActive && (
+            {index === 1 && isSatisfactionSurveyActive && isCookieEnabled && (
```

Comment:
Peut-être qu'on peut éviter d'utiliser un useState et un useEffect en faisant quelque chose comme ça ? :
```suggestion
            {index === 1 && isSatisfactionSurveyActive && window.localStorage && (
```

Reply:
en fait quand l'utilisateur désactive les cookie le code `window.localStorage` déclenche une exception d'ou le try catch au dessus